spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe 'Visit the user post index page', type: :feature do
+  before :each do
+    @user = User.create(
+      name: 'John Doe',
+      photo: 'https://randomuser.me/api/portraits/men/1.jpg',
+      bio: 'A passionate individual with interesting stories to share.'
+    )
+    @post1 = Post.create(author: @user, title: 'Post 1', text: 'This is the content of Post 1')
+    @post2 = Post.create(author: @user, title: 'Post 2', text: 'This is the content of Post 2')
+    @comment1 = Comment.create(post: @post1, author: @user, text: 'First comment on Post 1')
+    @comment2 = Comment.create(post: @post2, author: @user, text: 'First comment on Post 2')
+    @like1 = Like.create(post: @post1, user: @user)
+  end
+
+  it 'displays user information and posts on the index page' do
+    visit user_posts_path(@user)
+
+    expect(page).to have_selector('img[src=\'https://randomuser.me/api/portraits/men/1.jpg\']')
+    expect(page).to have_content 'John Doe'
+    expect(page).to have_content 'Number of posts: 2'
+    expect(page).to have_content 'Post 1'
+    expect(page).to have_content 'This is the content of Post 1'
+    expect(page).to have_content 'Post 2'
+    expect(page).to have_content 'This is the content of Post 2'
+    expect(page).to have_content 'First comment on Post 1'
+    expect(page).to have_content 'First comment on Post 2'
```

Comment:
- [ ] @Team Great work adding tests for the posts' index page, however on this test, you are supposed to check if `recent_comments` method is working correctly, that given more than five comments it will display five recent comments.